iAM-legacy/Simulation/Simulation.cs

```diff
@@ -1303,7 +1304,7 @@ public bool CreateSimulationTable(String strNetworkID, String strSimulationID)
 
             var numberColumns = m_listAttributes.Count * (Investment.AnalysisPeriod + 1);
 
-            var numberTables = Math.Ceiling(Convert.ToDouble(numberColumns) / 900);
+            var numberTables = Math.Ceiling(Convert.ToDouble(numberColumns) / 750);
```

Comment:
This magic number should be converted to a CONST which defines itself.

MAX_NUMBER_OF_MSSQL_COLUMNS or similar.